### PR TITLE
Basically add a label to operand namespace

### PR DESCRIPTION
**Pull request by rocrisp**
This enables OpenShift cluster monitoring for that namespace

### CODE REVIEW

File: internal/sub_reconcilers/reconcile_namespace.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+		Scheme: scheme,
+	}
+}
+
+func (r *NamespaceReconciler) Reconcile(ctx context.Context, bestie *petsv1.Bestie) (ctrl.Result, error) {
+	logInfo := types.NamespacedName{
+		Namespace: bestie.Namespace,
+		Name:      "NamespaceReconciler",
+	}
+	r.Log.WithValues("Namespace", logInfo)
```
Comment (skattoju) [inline internal/sub_reconcilers/reconcile_namespace.go:54]: Should probably say NamespaceReconciler instead of Namespace as it may be confusing

Comment (skattoju) [reply internal/sub_reconcilers/reconcile_namespace.go:54]: mmm actually this is more of a label reconciler no ?

File: internal/sub_reconcilers/reconcile_namespace.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+	}
+	if ns.ObjectMeta.Labels[namespaceLabel] != "true" {
+		ns.ObjectMeta.Labels[namespaceLabel] = "true"
+		err := r.client.Update(ctx, ns.DeepCopy())
+		if err != nil {
+			return ctrl.Result{}, err
+		}
+	}
+
+	// nsList := &corev1.NamespaceList{}
```
Comment (skattoju) [inline internal/sub_reconcilers/reconcile_namespace.go:69]: remove comments

**Review by skattoju — CHANGES_REQUESTED** [review]
some linter issues and questioning whether this should be called a label reconciler instead of a namespace reconciler .. what do you think ?

File: internal/sub_reconcilers/reconcile_namespace.go (hunk 1 above)
Comment (rocrisp) [reply internal/sub_reconcilers/reconcile_namespace.go:54]: Yep, you're right. Thank you for catching this, I was probably thinking too much on namespace when naming it.

**Comment by rocrisp** [comment]
fix #73 This pr is for the operand. Still need to implement for the operator

**Pull request merged**